double-conversion/double-conversion.h

```diff
@@ -294,13 +294,18 @@ class DoubleToStringConverter {
   // should be at least kBase10MaximalLength + 1 characters long.
   static const int kBase10MaximalLength = 17;
 
-  // Converts the given double 'v' to ascii. 'v' must not be NaN, +Infinity, or
-  // -Infinity. In SHORTEST_SINGLE-mode this restriction also applies to 'v'
-  // after it has been casted to a single-precision float. That is, in this
-  // mode static_cast<float>(v) must not be NaN, +Infinity or -Infinity.
+  // Converts the given double 'v' to digit characters. 'v' must not be NaN,
+  // +Infinity, or -Infinity. In SHORTEST_SINGLE-mode this restriction also
+  // applies to 'v' after it has been casted to a single-precision float. That
+  // is, in this mode static_cast<float>(v) must not be NaN, +Infinity or
+  // -Infinity.
   //
   // The result should be interpreted as buffer * 10^(point-length).
   //
+  // The digits are written to the buffer in the platform's charset, which is
+  // often UTF-8 (with ASCII-range digits) but may be another charset, such
+  // as EBCDIC.
```

Comment:
What about:

The digits are written to the buffer as ASCII digits. As such they can also be used in buffers with compatible charsets, such as UTF-8 or EBCDIC.

Reply:
What I was trying to say is that on an EBCDIC machine, you're not writing "ASCII digits", because EBCDIC and ASCII are mutually exclusive.  The literal `'0'` is equivalent to `0xF0` on EBCDIC machines but `0x30` on Latin-1/UTF-8 machines.

https://www.google.com/search?q=EBCDIC+vs+ASCII

If you meant "ASCII digits" in the sense to differentiate them from other numbering systems, like native arabic digits or bengali digits, calling them "Latin-script digits" or "western Arabic digits" might be more clear.  For example, maybe something like this:

> The digits are written as western Arabic numerals in the platform's default charset: for example, ASCII digits on UTF-8 machines or EBCDIC digits on certain IBM machines.

Reply:
Sorry for the late reply. (Exceptionally busy at work and home).
I have to admit I should have read your comment more carefully...

Not sure it's important to add this comment, and it might even be slightly be confusing, but I'm ok with it now.